ch.html

```diff
@@ -203,97 +187,69 @@ <h2 class="major"><font style="color:#ab2548;">/ </font>Prizes</h2>
         <!-- Four -->
         <section class="wrapper alt style1" id="four">
             <div class="inner">
-                <h2 class="major"><font style="color:#ab2548;">/ </font>The Alphatron Team</h2>
-                <p>Meet our main organizers!</p>
+                <h2 class="major"><font style="color:#ab2548;">/ </font>Das Alphatron-Team</h2>
+                <p>Lerne unsere Hauptorganisatoren kennen!</p>
                 <section class="features">
                     <article>
                         <img alt="" class="image" src="images/team/WeiszAgoston.jpg"/>
                         <h3 class="major">Agoston Weisz</h3>
-                        <h4>Main Event Organiser<br>‎</h4>
+                        <h4>Hauptorganisator<br>‎</h4>
                         <p class="dropdown-text" id="Weisz-Agoston">
-                            I am Weisz Agoston, a Senior Software Engineer at Google in Zurich.<br>
-                            I have spent my high-school years at the Fazekas Mihály Primary and Secondary Grammar School
-                            of Budapest, in a mathematics-focused class, from where I have gone to ELTE for a research
-                            MSc in mathematics.
-                            Középiskolás tanulmányaimat a Budapesti Fazekas Mihály Gyakorló Általános Iskola és
-                            I have won a bronze medal at the International Olympiad in Informatics, got first place at
-                            the Sapientia international informatics olympiad, and got third place twice on the KPI-open
-                            international team contest.
-                            Since 2013, I have received an award from the prime minister every year for my successful
-                            teaching - I have had at least one student on the international olympiads every year.<br>
-                            I am dedicated to providing the best possible education to as many people as I can.<br>
-                            In my free time, I cycle, ski and play football in the Google Zurich football team.
+                            Ich heisse Agoston Weisz und bin ein Senior Software Engineer bei Google in Zürich. Ich habe meine Schulzeit am Fazekas Mihály Grund- und Mittelgymnasium in Budapest in einer mathematisch orientierten Klasse verbracht, von wo aus ich an die ELTE gegangen bin, um einen Forschungs-Masterabschluss in Mathematik zu machen.<br>
```

Comment:
Same for heißen with ss instead of ß. Is that customary?

Reply:
Yes, Swiss Standard German always uses a double s (ss) instead of the eszett (ß).